From randy.dunlap@oracle.com  Mon Oct 20 16:05:51 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 13 Oct 2008 09:14:52 -0700
Subject: usb/gadget: fix kernel-doc warning
To: lud <linux-usb@vger.kernel.org>
Cc: dbrownell@users.sourceforge.net, gregkh <greg@kroah.com>
Message-ID: <20081013091452.61eb0c8c.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix kernel-doc warning, wrong parameter name listed:

Warning(lin2627-g3-kdocfixes//drivers/usb/gadget/config.c:183): No description found for parameter 'match'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/config.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/gadget/config.c
+++ b/drivers/usb/gadget/config.c
@@ -168,7 +168,7 @@ usb_copy_descriptors(struct usb_descript
  * usb_find_endpoint - find a copy of an endpoint descriptor
  * @src: original vector of descriptors
  * @copy: copy of @src
- * @ep: endpoint descriptor found in @src
+ * @match: endpoint descriptor found in @src
  *
  * This returns the copy of the @match descriptor made for @copy.  Its
  * intended use is to help remembering the endpoint descriptor to use
